packages/protocol/src/fabric/FabricManager.ts

```diff
@@ -253,7 +258,9 @@ export class FabricManager {
             );
         }
         this.#fabrics.set(fabricIndex, fabric);
-        await this.persistFabrics();
+        if (this.#fabrics) {
```

Comment:
should that not be "if storage"? fabrics is always true

Reply:
yes I think so